### PR TITLE
Smusand3

### PR DESCRIPTION
**Pull request by samsanders23**
Narrowing of security group outbound for add users proxy lambda.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -42,11 +42,11 @@ repos:
...
-- repo: https://github.com/aws-cloudformation/rain
-  rev: v1.2.0
-  hooks:
-  - id: cfn-format
-    files: \.template$
+# - repo: https://github.com/aws-cloudformation/rain
+#   rev: v1.2.0
+#   hooks:
+#   - id: cfn-format
+#     files: \.template$
```
Comment (rezabekf) [inline .pre-commit-config.yaml]: Hi Sam, any chance to fix your local development environment so we don't have to keep reverting this change?

File: cfn/add-users.template [hunk 2]
```diff
@@ -208,8 +208,15 @@ Resources:
...
-          IpProtocol: "-1"
+          Description: Allow all https outbound traffic
+          IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+        - DestinationSecurityGroupId: !Ref RDSSecurityGroup
+          Description: Allow db outbound traffic
+          IpProtocol: tcp
+          FromPort: 3306
+          ToPort: 3306
```
Comment (rezabekf) [inline cfn/add-users.template:219]: Does this change works for both Proxy and NoProxy clusters?

**Review by rezabekf — CHANGES_REQUESTED** [review]

File: cfn/add-users.template (hunk 2 above)
Comment (samsanders23) [reply cfn/add-users.template:219]: Should do, it was working during my test.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (samsanders23) [reply .pre-commit-config.yaml]: I can't find a way unfortunately

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**